packages/tailwindcss/src/css-functions.ts

```diff
@@ -1,19 +1,83 @@
 import { Features } from '.'
 import { walk, type AstNode } from './ast'
+import type { DesignSystem } from './design-system'
+import { withAlpha } from './utilities'
+import { segment } from './utils/segment'
 import * as ValueParser from './value-parser'
-import { type ValueAstNode } from './value-parser'
 
-export const THEME_FUNCTION_INVOCATION = 'theme('
+const functions: Record<string, (designSystem: DesignSystem, ...args: string[]) => any> = {
+  '--alpha': alpha,
+  '--spacing': spacing,
+  '--theme': theme,
+  theme,
+}
+
+function alpha(_designSystem: DesignSystem, value: string, alpha: string, ...rest: string[]) {
+  if (!value || !alpha) {
+    throw new Error(
+      `--alpha(…) requires 2 arguments, e.g.: \`--alpha(${value || 'var(--my-color)'}, ${alpha || '50%'})\``,
```

Comment:
Is it weird starting with `--alpha()`? Not sure what we've done in the past in these situations, but would something like this be better?

```suggestion
      `The --alpha(…) function requires 2 arguments, e.g.: \`--alpha(${value || 'var(--my-color)'}, ${alpha || '50%'})\``,
```

Reply:
I think both work, but `The … function` feels better, nice.